certbot/main.py

```diff
@@ -677,6 +687,8 @@ def main(cli_args=sys.argv[1:]):
         displayer = display_util.NoninteractiveDisplay(sys.stdout)
     elif config.text_mode:
         displayer = display_util.FileDisplay(sys.stdout)
+    elif config.dialog_mode:
```

Comment:
Merge `master` into this PR. The logic for choosing the display for renew moved [here](https://github.com/certbot/certbot/blob/master/certbot/cli.py#L337). This PR shouldn't have any changes on these lines and instead make the linked to code something like:
```
if self.verb == "renew" and not self.dialog_mode:
```